tutorials/raw/plot_visualize_raw.py

```diff
@@ -1,152 +1,165 @@
+# -*- coding: utf-8 -*-
 """
-.. _tut-viz-raw:
+Built-in plotting methods for Raw objects
+=========================================
 
-Visualize Raw data
-==================
+This tutorial shows how to plot continuous data as a time series, and how to
+plot the spectral density of continuous data.
 
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+As usual we'll start by importing the modules we need and loading some
+:ref:`example data <sample-dataset>`:
 """
-import os.path as op
-import numpy as np
 
+import os
 import mne
 
-data_path = op.join(mne.datasets.sample.data_path(), 'MEG', 'sample')
-raw = mne.io.read_raw_fif(op.join(data_path, 'sample_audvis_raw.fif'),
-                          preload=True)
-raw.set_eeg_reference('average', projection=True)  # set EEG average reference
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, preload=True, verbose=False)
 
 ###############################################################################
-# The visualization module (:mod:`mne.viz`) contains all the plotting functions
-# that work in combination with MNE data structures. Usually the easiest way to
-# use them is to call a method of the data container. All of the plotting
-# method names start with ``plot``. If you're using Ipython console, you can
-# just write ``raw.plot`` and ask the interpreter for suggestions with a
-# ``tab`` key.
+# We've seen in :ref:`a previous tutorial <tut-raw-class>` how to plot data
+# from a :class:`~mne.io.Raw` object using :mod:`matplotlib
+# <matplotlib.pyplot>`, but :class:`~mne.io.Raw` objects also have several
+# built-in plotting methods:
+#
+# - :meth:`~mne.io.Raw.plot`
+# - :meth:`~mne.io.Raw.plot_psd`
+# - :meth:`~mne.io.Raw.plot_psd_topo`
+# - :meth:`~mne.io.Raw.plot_projs_topomap`
+# - :meth:`~mne.io.Raw.plot_sensors`
+#
+# We'll discuss the first three here; :meth:`~mne.io.Raw.plot_projs_topomap` is
+# discussed in a later tutorial; :meth:`~mne.io.Raw.plot_sensors` is discussed
+# in :doc:`../misc/plot_sensor_locations`.
```

Comment:
I find kind of strange to have a page going through all the visualization methods of the raw container and that 2 of them are missing in this page.

Sure, details of those are explained in `plot_sensor_locations` what's in there and what can Id do. But maybe a small summuary (even if its just a plot) would be good to here. 

I think the way I would use that page is:
1 - I'm trying to figure out what raw is.
2 - visualizing it would help.
3 - I land on this page
4 - I don't read the page
5 - I scroll the entire page (I'm looking for plots, so I want to see which plots are to offer)
6 - back to the beginning 
7 - start reading.

but maybe my hypothetical workflow is off

Reply:
that makes sense.  Adding `plot_sensors` briefly and then pointing to the other tutorial, I can do that.  I'm more worried about `plot_projs_topomap` because I'm still thinking about these tutorials as sequential, so I've been trying to avoid mentioning or showing plots of projectors before they get introduced conceptually.  But maybe the lesson there is that a conceptual introduction to projectors needs to happen sooner (like as one of the tutorials in the "intro" section).  I'll make that my next priority, so that I can resolve some of these `TODO add link here` promises and not be constrained by what I think the reader doesn't know yet.